profiles/local-singularity/config.yaml

```diff
@@ -3,7 +3,7 @@ cores: 4
 printshellcmds: true
 rerun-incomplete: true
 use-singularity: true
-singularity-args: "--bind $PWD/../../../zarp"
+singularity-args: "--bind $PWD/../../../zarp --bind $TMPDIR"
```

Comment:
I think this would work:

```console
--bind $PWD/../../../zarp:$TMPDIR
```

But I'm not sure what happens if `$TMPDIR` is unset, or set to an empty string. If that doesn't cause a problem, I don't see what speaks against including it.

Now, what speaks _for_ including it is that it apparently works. However, I don't quite understand why there isn't `$TMP` or `$TMPDIR` set inside the container. And why it works when `$TMPDIR` is mounted but not set? It's all a bit weird...

Maybe somebody else has an idea. And yes, how is this problem addressed in ZARP?

@mkatsanto @dominikburri?

Reply:
I looked into this a bit but could not yet figure out how it is connected, indeed funny that binding but not setting solves the problem. Probably a lot going on in the background with implicit/default settings. 

In any case, should the same fix also be applied to the slurm-singularity profile?